scripts/pull.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Why is this file in the diff? I don't want this merged, cos that would revert pull.sh to a previous version.